internal/schema/1.5/root.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/zclconf/go-cty/cty"
+
+	v1_4_mod "github.com/hashicorp/terraform-schema/internal/schema/1.4"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
+)
+
+func ModuleSchema(v *version.Version) *schema.BodySchema {
+	bs := v1_4_mod.ModuleSchema(v)
+	bs.Blocks["import"] = &schema.BlockSchema{
+		Description: lang.Markdown("Import resources into Terraform to bring them under Terraform's management"),
+		Body: &schema.BodySchema{
+			HoverURL: "https://developer.hashicorp.com/terraform/language/import",
```

Comment:
We may want to eventually make this version-specific URL, but that would apply to any existing and future links.